docs/EN_US/DataPatterns/DataPatternsMods/DataPatternsIntro.xml

```diff
@@ -1,18 +1,60 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE sect1 PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+<!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
 "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
 <chapter id="DPIntro">
   <title><emphasis>Data Patterns Introduction</emphasis></title>
 
-  <para>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-  eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
-  veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
-  commodo consequat.</para>
+  <para>Data Patterns is a Library and ECL bundle that primarily serves as a
+  data profiling tool to give an analysis of a dataset in HPCC.</para>
 
```

Comment:
"Data Patterns" is most precisely a bundle containing several functions/features.  Colloquially, people use "Data Patterns" to mean "data profiling" which is one feature of Data Patterns.  Even ECL Watch does this.

It may be worthwhile calling that out here, and further stating that in many places within this document "Data Patterns" is used to mean "data profiling".  The alternative would be changing an awful lot of text, which may be unpalatable.